src/Todo.tsx

```diff
@@ -0,0 +1,39 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+import "./Todo.css";
+
+interface ITodo {
+    id: number;
+    summary: string;
+}
+
+
+function Todo() {
+    const [todos, setTodos] = useState<ITodo[]>([]);
+    const [newTodos, setNewTodos] = useState<string>("");
+
+    function onSubmitForm(event: FormEvent<HTMLFormElement>) {
+        event?.preventDefault();
```

Comment:
여기서는 옵셔널 체이닝이 필요 없을 거 같아요!

```suggestion
        event.preventDefault();
```